packages/react/hover-card/src/HoverCard.tsx

```diff
@@ -277,12 +328,19 @@ const HoverCardContentImpl = React.forwardRef<
       <PopperPrimitive.Content
         {...popperScope}
         {...contentProps}
+        onPointerDown={composeEventHandlers(contentProps.onPointerDown, () => {
+          setContainTextSelection(true);
+          context.isTextSelectingRef.current = true;
```

Comment:
Do we need the extra ref if we have a state already that tells us we're containing a selection?
I know it's currently lower down, but we could lift it?

Reply:
We need this to handle the "has a selection" case which prevents closing if there's a selection after pointerUp and pointer leaves the card.

It also informs parent layers that a child has a selection and to not close when "pointerLeave" bubbles